pkgs/tools/package-management/akku/default.nix

```diff
@@ -1,7 +1,7 @@
-{ lib, newScope, stdenv, fetchurl }:
+{ lib, newScope, fetchurl }:
 lib.makeScope newScope (self: rec {
 
-  fetchAkku = { name, url, sha256, ... }:
+  fetchAkku = { url, sha256, ... }:
```

Comment:
While we are at it, since `fetchAkku` is nothing special, should we just use `fetchurl` in its place?